apple/internal/watchos_rules.bzl

```diff
@@ -644,7 +651,7 @@ def _watchos_extension_impl(ctx):
             label_name = label.name,
             platform_prerequisites = platform_prerequisites,
             predeclared_outputs = predeclared_outputs,
-            product_type = rule_descriptor.product_type,
+            product_type = product_type,
```

Comment:
Also needed above in `process_entitlements`?

Reply:
Yes indeed!